src/test/scala/com/github/symcal/ScalaCompiledFunctionSpec.scala

```diff
@@ -0,0 +1,50 @@
+package com.github.symcal
+
+import org.scalatest.{FlatSpec, Matchers}
+
+class ScalaCompiledFunctionSpec extends FlatSpec with Matchers {
+
+  behavior of "scala compiled function"
+
+  it should "compile and run simple functions" in {
+    val func = ScalaCompiledFunction.compile[Double ⇒ Double]("x: Double" -> "x / 2")
```

Comment:
Yes, it is required because now we parameterize `ScalaCompiledFunction.compile[...]` and it doesn't work without ": Double".